### PR TITLE
[ADD] brand_stock_account

### PR DESCRIPTION
**Pull request by acsonefho**
New module `brand_stock_account` to auto-fill the brand on the move entry (account.move) auto-generated with inventory valuation, when the picking is validated.

### CODE REVIEW

**Review by adrienpeiffer — APPROVED** [review]

File: brand_stock_account/models/account_move.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    @api.model
+    def _fill_brand_id(self, values):
+        values = values.copy()
```
Comment (sbejaoui) [inline brand_stock_account/models/account_move.py:11]: why copy values?

File: brand_stock_account/models/account_move.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        return values
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        """
+
+        :param vals_list: list of dict
+        :return: self recordset
+        """
+        list_vals = [self._fill_brand_id(vals) for vals in vals_list]
```
Comment (sbejaoui) [inline brand_stock_account/models/account_move.py:28]:
```suggestion
        # FIXME: 
        # we intercept the creation of the account.move to add the brand
        # due to non-inheritable code in `stock.move, _create_account_move_line` 
        list_vals = [self._fill_brand_id(vals) for vals in vals_list]
```
a comment here will help in future migration to improve the code

File: brand_stock_account/models/account_move.py (hunk 1 above)
Comment (acsonefho) [reply brand_stock_account/models/account_move.py:11]: It's a good practice, never update `dict` given. In some case you could have issues.

Comment (sbejaoui) [reply brand_stock_account/models/account_move.py:11]: I don't see the benefit here, can you, please, point a link that explains it, for my general knowledge

Comment (acsonefho) [reply brand_stock_account/models/account_move.py:11]: I don't have any link, it's not documented. But in any case, this is not a blocking point about PR approval.

Comment (sbejaoui) [reply brand_stock_account/models/account_move.py:11]: Yes indeed

**Review by sbejaoui — APPROVED** [review]
LGTM, thanks

**Pull request merged**